owasp-top10-2017-apps/a1/copy-n-paste/docs/ATTACK.md

```diff
@@ -65,10 +66,16 @@ $ sqlmap -r postRequest.txt
 
 <img src="attack-3.png" align="center"/>
 
-After understanding how this database is structured, an attacker could use the following command to retrieve sensitive information from the it:
+After understanding how this database is structured, an attacker could use the following command to retrieve database details:
+
+```sh
+$ sqlmap -r postRequest.txt --tables
+```
+
+And then retrieve sensitive information from the it:
```

Comment:
Nice contribution! Could you just fix this typo : `And then retrieve sensitive information from the it:` to `And then retrieve sensitive information from it: `

Reply:
Done!